### PR TITLE
Update lazysets.jl

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: _literate/howTo/lazysets.jl [hunk 1]
```diff
@@ -28,7 +28,8 @@ S = @constraint x^2+y^2 + 3*sin(x) + 5*sin(y) <= 1.0
 X = IntervalBox(-10..10, 2) # our starting box
 paving = pave(S, X, 0.02)
 
-# choose octagon directions
+# We will choose octagon directions, meaning that the directions normal to the
+# overapproximating a polyhedron are parallel to those of an octagon (not restricted to two dimensions):
```
Comment (mforets) [inline _literate/howTo/lazysets.jl]:
```suggestion
# overapproximating polyhedron are parallel to those of an octagon (this method is not restricted to two dimensions):
```

**Comment by mforets** [comment]
ready for review :smiling_face_with_three_hearts:

**Comment by lucaferranti** [comment]
Looks good! a small comment, I think in the [last image](https://raw.githubusercontent.com/JuliaIntervals/juliaintervals.github.io/gh-pages/assets/pages/howTo/lazysets/code/output/pavingPolyhedralRefined.svg) the gray box around the zoomed area cannot be seen very clearly, is it possible to make it darker?

**Comment by mforets** [comment]
yes, sure! i added one commit to make it a bit darker. feel free to edit the figs as will

**Pull request merged**